system/jlib/jsocket.cpp

```diff
@@ -1581,6 +1581,8 @@ void CSocket::udpconnect()
     if (sock == INVALID_SOCKET) {
         THROWJSOCKEXCEPTION(ERRNO());
     }
+    if (sockSize > 0)
+        set_send_buffer_size(sockSize);
```

Comment:
Probably does not matter for udp, but technically, socket buffer sizes should be set before connect()